### PR TITLE
feat: Default nested limit

### PR DESCRIPTION
**Pull request by Harshith46**
This is for setting the default collection item limit. The default limit will be set to all the child or nested collection items.
if storyLimits is not passed, it will fall back to this default limit. If we don't pass the default limit, it will fall back to 40

https://github.com/quintype/ace-planning/issues/379

Dependencies - https://github.com/quintype/quintype-node-backend/pull/50

Documentation  -  https://developers.quintype.com/malibu/tutorial/adding-default-limit-to-collections.html

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ags1773 — APPROVED** [review]

File: app/server/load-data.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ export function loadData(pageType, params, config, client, { host, next, domainS
   function _loadData() {
     switch (pageType) {
       case PAGE_TYPE.HOME_PAGE:
-        return loadHomePageData(client, config, params.collectionSlug);
+        return loadHomePageData(client, config, params.collectionSlug, publisherAttributes);
```
Comment (nandap4790) [inline app/server/load-data.js:56]: only the options can be passed right ?

File: config/publisher.yml [hunk 2]
```diff
@@ -23,3 +23,10 @@ publisher:
     open_in_new_tab: false
     pages: [home-page, section-page, tag-page, search-page, story-page, catalog-page, static-page, form-page]
   should_use_collection: true
+  options:
```
Comment (nandap4790) [inline config/publisher.yml:26]: can we have a more meaningful naming?

**Review by nandap4790 — CHANGES_REQUESTED** [review]

File: app/server/load-data.js (hunk 1 above)
Comment (Harshith46) [reply app/server/load-data.js:56]: yes

File: config/publisher.yml (hunk 2 above)
Comment (Athira001) [reply config/publisher.yml]: Done

**Review by nandap4790 — APPROVED** [review]

**Pull request merged**